ee/experimental/note_utils.py

```diff
@@ -0,0 +1,76 @@
+import heapq
```

Comment:
note_utils?

Reply:
don't think this is a util, but some model definition

Reply:
Pls share a proposal on how to name it :)